docs/adding-a-library.md

```diff
@@ -0,0 +1,312 @@
+# Integrating a library within the hub
+
+The Hugging Face hub aims to facilitate the sharing of machine learning models, checkpoints, and artifacts. 
+This endeavor starts with the integration of its tool stack within downstream libraries, and we're happy to announce 
+the fruitful collaboration between Hugging Face and SpaCy, AllenNLP, Timm, among many other incredible libraries.
+
+> For a full list of supported libraries see [the following table](https://github.com/huggingface/huggingface_hub/blob/96e49483ace685cf526492b248ba3341bc1b1564/docs/libraries.md).
+
+We believe the model hub is a step in the correct direction for several reasons. It offers:
```

Comment:
```suggestion
We believe the Hub is a step in the correct direction for several reasons. It offers:
```

I would avoid mentioning **model hub**, since we're also looking for dataset integration